web/angular-wallet/src/app/setup/account/account.service.ts

```diff
@@ -399,7 +399,8 @@ export class AccountService {
       const http = HttpClientFactory.createHttpClient(environment.activatorServiceUrl);
       const payload = {
         account: account.account,
-        publickey: account.keys.publicKey
+        publickey: account.keys.publicKey,
+        ref: `phoenix-${environment.version}`
```

Comment:
this is an extension to track activation.... having setup logz.io account and feeding it through the account activation service